scoreboard/wsgi.py

```diff
@@ -13,11 +13,12 @@
 # limitations under the License.
 
 from scoreboard import main
-from scoreboard import rest
-from scoreboard import views
 
 app = main.get_app()
 main.load_config_file(app)
 
+from scoreboard import rest
```

Comment:
More specifically, importing `rest` imports `attachment/__init__.py` which expects the config values to be loaded